Apps/APN-UIKit/APN UIKit/APN UIKit/Router/DashboardRouter.swift

```diff
@@ -11,9 +11,11 @@ class DashboardRouter: DashboardRouting {
     weak var dashboardViewController: DashboardViewController?
 
     func routeToLogin() {
-//        let viewController = LoginViewController.newInstance()
-//        dashboardViewController?.navigationController?.pushViewController(viewController, animated: true)
-        dashboardViewController?.navigationController?.popToRootViewController(animated: true)
+        guard let _ = dashboardViewController?.navigationController?.popToRootViewController(animated: true) else {
+            let viewController = LoginViewController.newInstance()
+            dashboardViewController?.navigationController?.pushViewController(viewController, animated: true)
+            return
+        }
```

Comment:
```suggestion
        let didNavigateBackToLogin = dashboardViewController?.navigationController?.popToRootViewController(animated: true)
        
        if !didNavigateBackToLogin {
        dashboardViewController?.navigationController?.pushViewController(LoginViewController.newInstance(), animated: true)
        }
```

I found the `guard` to make the code harder to follow then an `if` in this function. The code suggestion was easier to read, IMO.

Reply:
To me, it is clear what `guard let` statement does but if you feel it is confusing you, then I am going ahead and accepting your suggestion.